website/docusaurus.config.js

```diff
@@ -5,8 +5,7 @@ const editUrl = 'https://github.com/yewstack/yew/blob/master/website/'
 /** @type {import('@docusaurus/types').DocusaurusConfig} */
 module.exports = {
     title: 'Yew',
-    tagline:
-        'Yew is a modern Rust framework for creating multi-threaded front-end web apps with WebAssembly.',
+    tagline: 'A modern Rust framework for creating web applications.',
```

Comment:
```suggestion
    tagline: 'A modern Rust framework for creating safe and performant web applications.',
```

Whilst I think the new slogan is good as-is, I think it might be better to include some description of the key features similar to the Rust slogan.

Reply:
My suggestion:

_Yew: A framework for creating reliable and efficient web apps_

In summary of #2378, the slogan should focus on what the user can *do* with Yew, not what Yew *is*.

The Rust slogan previously used words like _fast_ and _safety_, which have since been replaced with the phrasing _reliable and efficient_. I think this exact phrasing is also suitable for Yew, given its attractive association with the Rust language, and the fact that _safety_ in the context of web is a vague term often conflated with web _security_.

It may seem strange to replace _modern Rust framework_  with just _frameowrk_, but this focuses more on what the user can *do*. Mentions of Rust and WebAssembly are readily found in the opening paragraph or about section.

//

This comment and #2378 were inspired by [_A new look for the rust-lang.org_](https://blog.rust-lang.org/2018/11/29/a-new-look-for-rust-lang-org.html) on the Rust Blog.